packages/admin-ui/src/calendar/utils.ts

```diff
@@ -67,9 +67,8 @@ export function generateDaysInMonthArray(
     .reduce((weeks: Date[][], _, weekIndex) => {
       const daysInWeek = [0, 1, 2, 3, 4, 5, 6].reduce(
         (days: Date[], dayIndex) => {
-          const day = weekIndex * 7 + dayIndex - monthStartsAt + 2
-          const utcDate = toUTCString(new Date(year, month, day))
-          const cellDate = new Date(utcDate)
+          const day = weekIndex * 7 + (dayIndex - monthStartsAt) + 1
```

Comment:
What is `7` and `1`. Could you extract to var?

Reply:
I'm inferring that 7 is the 7 days from a week, and that 1 is the first day of a week (monday or sunday depending on the locale). The actual fix relies on these lines as far as I'm understanding!